### DISCUSSION TITLE
Ability to use booksmarks/views

### DISCUSSION THREAD

**Discussion by Attacler** [directus/directus#7970] [Feature Requests]
Hey everyone,

My idea is to make bookmarks selectable in the collection overview.
So for example a dropdown on the topleft which allows you to select a bookmark to be used.

An example senario:
You have a collection called invoices.
For this collection, you want to make sure that all paid invoices are visible so you select the "paid invoices" bookmark.
And the same for unpaid.
Basicly predefined filters (per user/role).

What do you guys think about this?

**Comment by rijkvanzanten** [comment]
How is this different from the current bookmarking functionality? 🤔

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
